sonargraphintegration.properties

```diff
@@ -4,20 +4,26 @@ category=External Analyzers
 description=Provides support for Sonargraph 8+'s architecture governance features, accompanied by metrics about cyclic dependencies and other structural aspects.
 homepageUrl=https://github.com/sonargraph/sonar-sonargraph-integration
 devVersion=
-publicVersions=5.1.0
-archivedVersions=1.1.2,1.1.3,1.1.4,2.0.1,2.1.5,2.1.7,3.1.0,4.0.0,4.1.1,4.1.2,5.0.0
+publicVersions=5.2.0
+archivedVersions=1.1.2,1.1.3,1.1.4,2.0.1,2.1.5,2.1.7,3.1.0,4.0.0,4.1.1,4.1.2,5.0.0,5.1.0
 privateVersions=
 
 defaults.mavenGroupId=com.hello2morrow
 defaults.mavenArtifactId=sonar-sonargraph-integration
 
+5.2.0.description=Compatibility with Sonargraph 10.4.
+5.2.0.sqVersions=[7.9,LATEST]
+5.2.0.date=2020-09-28
+5.2.0.changelogUrl=https://github.com/sonargraph/sonar-sonargraph-integration/releases/download/Release-5.2/release-notes.txt
+5.2.0.downloadUrl=https://github.com/sonargraph/sonar-sonargraph-integration/releases/download/Release-5.2/sonar-sonargraph-integration-5.2.jar
+
+#====================
 5.1.0.description=Additional built-in quality profile. Reworked storage of custom metrics.
 5.1.0.sqVersions=[7.9,LATEST]
```

Comment:
Only one version can be compatible with `LATEST`. Set this end range to the current version: 8.4.2, or 8.4.*.
That will make sure the Marketplace prompts your users on 8.4.* to upgrade to your latest version.

Reply:
Thanks for the hint! Updated accordingly.